components/announcement_bar/default_announcement_bar/announcement_bar.tsx

```diff
@@ -134,7 +134,7 @@ export default class AnnouncementBar extends React.PureComponent<Props> {
                     placement='bottom'
                     overlay={announcementTooltip}
                 >
-                    <span>
+                    <div className='announcement-bar__text'>
```

Comment:
This isn't strictly necessary, but I wanted to fix the type (since spans can't contain a lot of the elements that show up in Markdown text) and use a more specific selector than just `> span`